### ISSUE TITLE
Window Closing issue

### ISSUE THREAD

**Issue by Togela** [pyvista/pyvista#1018]
Hello everyone,
I'm using spyder with pyvista. When I'm plotting consecutively in external windows (via `notebook=False`) after the first window all the windows instantly close after opening. This could be a possible duplicate of #501 . Rolling back to version 0.25.2 fixes the bug. 
-----

**To Reproduce**

```py
import pyvista as pv
spherebig = pv.Sphere(1, (0,0,0))
spherebig.plot(notebook=False)
spheresmall = pv.Sphere(0.5, (0,0,0))
spheresmall.plot(notebook=False)
allspheres = spherebig.boolean_add(spheresmall)
allspheres.plot(notebook=False)
```

**System Information:**
```txt
--------------------------------------------------------------------------------
  Date: Tue Nov 17 07:47:23 2020 Mitteleuropäische Zeit

                OS : Windows
            CPU(s) : 32
           Machine : AMD64
      Architecture : 64bit
               RAM : 127.9 GB
       Environment : Jupyter
        GPU Vendor : NVIDIA Corporation
      GPU Renderer : Quadro RTX 4000/PCIe/SSE2
       GPU Version : 4.5.0 NVIDIA 430.86

  Python 3.8.3 (default, May 19 2020, 06:50:17) [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.27.2
               vtk : 9.0.1
             numpy : 1.19.4
           imageio : 2.9.0
           appdirs : 1.4.4
            scooby : 0.5.6
            meshio : 4.3.4
        matplotlib : 3.3.3
             PyQt5 : 5.12.3
           IPython : 7.19.0
             scipy : 1.5.4
              tqdm : 4.46.0
--------------------------------------------------------------------------------
```

**Comment by ZandreeTresvalles** [comment]
Having the same issue, I tried downgrading to vtk==8.1.2 and python==3.7.9 pyvista== 0.27.3

**Comment by akaszynski** [comment]
Having this issue as well.  I'll have this patched in the next release `pyvista==0.27.4`.
